python/ray/tests/BUILD

```diff
@@ -354,6 +353,25 @@ py_test_module_list(
     ],
 )
 
+##### Begin 'minimal installation' tests #####
+
+py_test_module_list(
```

Comment:
What's the overhead of launching a new instance? Is it more economical compared to sharing an instance with other tests?

Reply:
there's no overhead, they are just logical groupings